decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
```

Comment:
- [ ] I will implore you to use `attr_accessor :nameable` so that you can read and write it.